### ISSUE TITLE
Bug in the code!

### ISSUE THREAD

**Issue by Charliearg-ui** [ljhowell/machine-learning_CHD#7]
Guys - found a bug in the code, its in the Logistic Regression module, can someone help me take a look? Thanks!!

**Comment by Charliearg-ui** [comment]
just for video purposes x

**Comment by ljhowell** [comment]
Yes sure, please give some more details and steps for replication and I am happy to have a look now

**Comment by Charliearg-ui** [comment]
Brilliant - thank you! Let me just check it again, and will write out some instructions of what i did!
